larq_zoo/dorefanet.py

```diff
@@ -0,0 +1,183 @@
+import tensorflow as tf
+import larq as lq
+from larq_zoo import utils
+from zookeeper import registry, HParams
+
+
+@registry.register_model
+def dorefa_net(hparams, input_shape, num_classes, input_tensor=None, include_top=True):
+    def conv_block(x, features, kernel_size, strides=1, pool=False, padding="same"):
```

Comment:
I'd prefer the following to keep wording consistent:
```suggestion
    def conv_block(x, filters, kernel_size, strides=1, pool=False, padding="same"):
```